src/brain.coffee

```diff
@@ -398,7 +398,8 @@ class Brain
     else
       # For everything else, strip all non-alphanumerics
       msg = utils.stripNasties(msg, @utf8)
-
+	# cut leading and trailing blanks
+	msg = utils.trimWhitespace(msg);
```

Comment:
In coffeescript, indentation levels are important. These lines should be indented to the same level as the `return` statement.

Also, `String.prototype.trim()` is already built into the language, so this line can be replaced with:

```coffeescript
msg = msg.trim()
```